README.md

```diff
@@ -29,9 +29,10 @@ To include the suite in your project, you only need to invoke its functions from
 -include_lib("mixer/include/mixer.hrl").
 -mixin([ktn_meta_SUITE]).
 
--export([init_per_suite/1]).
+-export([init_per_suite/1, end_per_suite/1]).
 
 init_per_suite(Config) -> [{application, your_app} | Config].
+end_per_suite(_) -> ok.
```

Comment:
It does not need to.
> Type spec of end function is
[relaxed](https://github.com/erlang/otp/blob/OTP-19.3.4/lib/common_test/doc/src/common_test_app.xml#L251-L257),
and examplary return value is
[`ok`](https://github.com/erlang/otp/blob/OTP-19.3.4/lib/common_test/doc/src/example_chapter.xml#L95).